### ISSUE TITLE
Document language missing

### ISSUE THREAD

**Issue by shiffman** [shiffman/shiffman.net#63]
<img width="347" alt="screen shot 2018-10-19 at 2 25 50 pm" src="https://user-images.githubusercontent.com/191758/47236679-e97d3c80-d3aa-11e8-80b0-ac7d3e6bb6e3.png">

From: https://shiffman.net/a2z/

**Comment by phucduong86** [comment]
Hi Daniel,
Can I work on this issue? 
Thank you.

**Comment by shiffman** [comment]
Yes, go for it!
